ui/src/App.js

```diff
@@ -142,6 +142,8 @@ const Admin = (props) => {
 }
 
 const AppWithHotkeys = () => {
+  let language = localStorage.getItem('locale');
```

Comment:
We need to have a default here:
```
      const language = localStorage.getItem('locale') || 'en';
```

Reply:
Okay sure wait a moment

Reply:
Yeah, have changed that and committed.
please check once.